docs/features/models/NER.ipynb

```diff
@@ -0,0 +1,671 @@
+   "source": [
+    "After [installing](#2.-Get-started-with-the-model) the model, build it from the config and predict."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "fnHw4a6C9l_4"
+   },
+   "outputs": [],
+   "source": [
+    "from deeppavlov import build_model\n",
+    "\n",
+    "ner_model = build_model('ner_ontonotes_bert_torch', download=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Input**: List[sentences]\n",
+    "\n",
+    "**Output**: List[tokenized sentences, corresponding NER-tags]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "QFXXQf3Q9l_6",
+    "outputId": "a6087769-fafc-4821-8254-465bc98b9886"
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[[['Bob', 'Ross', 'lived', 'in', 'Florida'],\n",
+       "  ['Elon', 'Musk', 'founded', 'Tesla']],\n",
+       " [['B-PERSON', 'I-PERSON', 'O', 'O', 'B-GPE'],\n",
+       "  ['B-PERSON', 'I-PERSON', 'O', 'B-ORG']]]"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "ner_model(['Bob Ross lived in Florida', 'Elon Musk founded Tesla'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "RLeAxnlu9l_7"
+   },
+   "source": [
+    "## 3.2 Predict using CLI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also get predictions in an interactive mode through CLI."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! python -m deeppavlov interact ner_ontonotes_bert_torch -d"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`-d` is an optional download key (alternative to `download=True` in Python code). The key `-d` is used to download the pre-trained model along with embeddings and all other files needed to run the model. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Or make predictions for samples from *stdin*."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! python -m deeppavlov predict ner_ontonotes_bert_torch -f <file-name>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 4. Evaluate "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are two metrics that are used to evaluate a NER model in DeepPavlov:\n",
+    "\n",
+    "`ner_f1` is measured on the entity-level (actual text spans should match exactly)\n",
+    "\n",
+    "`ner_token_f1` is measured on a token level (correct tokens from not fully extracted entities will still be counted as TPs)"
```

Comment:
"`ner_token_f1` is measured on a token level (correct tokens from not fully extracted entities will still be counted as TPs (true positives))"